src/net.cpp

```diff
@@ -1635,10 +1638,15 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
             }
 
             if (add_fixed_seeds_now) {
+                std::vector<CAddress> seed_addr{ConvertSeeds(Params().FixedSeeds())};
```

Comment:
nit: i think plural (`seed_addrs`) would make more sense, in the latter calls to this vector.